<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1600</width>
    <height>901</height>
   </rect>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1300</width>
      <height>821</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="directionGroupBox">
    <property name="geometry">
     <rect>
      <x>1330</x>
      <y>10</y>
      <width>251</width>
      <height>211</height>
     </rect>
    </property>
    <property name="title">
     <string>Direction Editor</string>
    </property>
    <widget class="QRadioButton" name="riverButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>River</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="grassButton">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>30</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Grass</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="populationButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>151</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Population Center</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="guidanceButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Field Guider</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="groundButton">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>30</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Ground</string>
     </property>
    </widget>
    <widget class="QSlider" name="weightSlider">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>150</y>
       <width>211</width>
       <height>22</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="weightLabel">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>170</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>weight: 1.0</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1600</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLoadTerrain"/>
    <addaction name="actionSaveTerrain"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionLoadTerrain">
   <property name="text">
    <string>Load Terrain</string>
   </property>
  </action>
  <action name="actionSaveTerrain">
   <property name="text">
    <string>Save Terrain</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
